tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "At least one course in the Department of Classics or cross-listed as CLA. A second course in addition to the prerequisite is strongly recommended, as well, but this course can count as one of the six departmentals focusing on classical civilization. A freshman seminar on a classical subject may count as a prerequisite, but may not be used as a departmental.\n\nApplicants to this track must submit a proposed plan of study, worked out in consultation with the DUS, by the end of Fall term in their junior year. Given the range of possible interests (e.g., Greek and Roman culture, Late Antique or Byzantine/Mediaeval literature, the reception of antiquity in later periods and cultures), each applicant may bring to the study of the ancient world, there is no set list of fields of concentration, and faculty members can give additional guidance in preparing a track of study.",
+              "course_list": [
+                "CLA *"
+              ]
+            },
+            {
+              "name": "Ancient Greek or Latin",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Each student must successfully complete either Ancient Greek or Latin to the level of 108 or achieve an equivalent level of knowledge, as demonstrated through test scores (SAT, AP), a placement exam, or coursework (including through a summer language program). However, at least one language course must be taken at Princeton.",
+              "course_list": [
+                "LAT 108",
+                "CLG 108"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nThe specific courses to be taken by each student must form a coherent program of study. Whatever the individual concentration, each student's program must contain eight departmentals and the Junior Seminar.",
+              "req_list": [
+                {
+                  "name": "Classical Civilization",
+                  "max_counted": null,
+                  "min_needed": null,
+                  "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nSix of these courses must focus in whole or in part on classical civilization or its influence. At least three of these courses must be taught in the Department of Classics (CLA, CLG, or LAT). Courses taken during the freshman and sophomore years beyond the prerequisite may count toward this requirement if they are appropriate to the student's overall program.",

```

Update the editor's note
```suggestion
                  "explanation": "<i>Editor's Note: Concentrators must get departmental approval for these courses, and TigerPath does not verify this.</i>\n\nSix of these courses must focus in whole or in part on classical civilization or its influence. At least three of these courses must be taught in the Department of Classics (CLA, CLG, or LAT). Courses taken during the freshman and sophomore years beyond the prerequisite may count toward this requirement if they are appropriate to the student's overall program.",
```